src/scs/patching-automation.md

```diff
@@ -0,0 +1,36 @@
+# Patch Management Automation
+
+Patch management automation increases complex software development efficiency concurrently ensuring high level of vulnerability remediation in a reasonable time frame.
+
+The process is:
+
+  - Build the package
+  - Scan for vulnerabilities
+  - If no new vulnerabilities are discovered the package undergoes testing followed by the developer review
+  - Check for update availability and upgrade
+  - Rebuild the package
+  - Rescan and send it to the automatic testing, followed by the developer review
+  
+![AutomaticPatching](../img/autopatching.drawio.png)
```

Comment:
Let's change this a little bit.

To insert images in Markdown, we use the following markdown syntax (I will also add this to the style guide):
![alt text for screen readers](/path/to/image.png "Text to show on mouseover")

So, let's write here:
![Patch Management Automation](../img/autopatching.drawio.png "Automated Patch Management Solution")